packages/react-components/react-breadcrumb/src/components/BreadcrumbLink/useBreadcrumbLinkStyles.ts

```diff
@@ -52,11 +65,33 @@ const useStyles = makeStyles({
   },
 });
 
+const useIconStyles = makeStyles({
+  base: {
+    fontSize: `var(${breadcrumbCSSVars.breadcrumbIconSizeVar})`,
+    height: `var(${breadcrumbCSSVars.breadcrumbIconSizeVar})`,
+    lineHeight: `var(${breadcrumbCSSVars.breadcrumbIconLineHeightVar})`,
+    width: `var(${breadcrumbCSSVars.breadcrumbIconSizeVar})`,
+  },
+  small: {
+    [breadcrumbCSSVars.breadcrumbIconSizeVar]: '12px',
+    [breadcrumbCSSVars.breadcrumbIconLineHeightVar]: tokens.lineHeightBase200,
+  },
+  medium: {
+    [breadcrumbCSSVars.breadcrumbIconSizeVar]: '16px',
+    [breadcrumbCSSVars.breadcrumbIconLineHeightVar]: tokens.lineHeightBase400,
+  },
+  large: {
+    [breadcrumbCSSVars.breadcrumbIconSizeVar]: '20px',
+    [breadcrumbCSSVars.breadcrumbIconLineHeightVar]: tokens.lineHeightBase600,
```

Comment:
Not sure that I understand why CSS variables are needed there, can it be applied directly via styles?

Reply:
In case if partners want to override styling for the icons. Isn't it easier for them to override using CSS variables? If not, I'll just use styles

Reply:
Let's use styles as it's aligned with other components.